src/app/app.component.html

```diff
@@ -28,3 +28,4 @@
 
 <notifications></notifications>
 <dialog-base></dialog-base>
+<video-player></video-player>
```

Comment:
I can see that most of the code just duplicates what dialogs do. Why cannot we make use of existing dialogs mechanism? We would then have videos under `(dialog: video/{videoId})` path and we would be able to simply navigate to them by using `DialogService`.

Reply:
I wanted separate component for the dialog window, I did not want to reuse the existing one. And I wanted to keep the possibility for future when we replace the standard dialogs. Is it ok?

Reply:
I simply cannot see any advantage of this approach. If we decide to replace dialog mechanism (as you suggest), we will need to change it on two places since the code will be duplicated. Or do I miss anything?

Reply:
And this is exactly it - I don’t think we will be replacing the video player one as it will keep the default look’n’feel. Yes/No dialog is usually a very different from a specialized one... Is it ok?